packages/search-in-workspace/src/browser/search-in-workspace-widget.tsx

```diff
@@ -435,15 +452,27 @@ export class SearchInWorkspaceWidget extends BaseWidget implements StatefulWidge
     protected renderReplaceAllButtonContainer(): React.ReactNode {
         // The `Replace All` button is enabled if there is a search term present with results.
         const enabled: boolean = this.searchTerm !== '' && this.resultNumber > 0;
+
+        const replace = () => {
+            if (enabled) {
+                this.resultTreeWidget.replace(undefined);
+            }
+        };
+
         return <div className='replace-all-button-container'>
             <span
                 title='Replace All'
+                tabIndex={0}
                 className={`replace-all-button${enabled ? ' ' : ' disabled'}`}
                 onClick={() => {
-                    if (enabled) {
-                        this.resultTreeWidget.replace(undefined);
+                    replace();
+                }}
+                onKeyPress={e => {
+                    if (e.key === 'Enter') {
```

Comment:
@akosyakov I should have also added support for  `Space` key here as mentioned in https://github.com/eclipse-theia/theia/pull/8079#discussion_r446644852 but there is an issue shown in the below Gif(sorry I don't know how to record which key is pressed):

![space](https://user-images.githubusercontent.com/46004116/85947914-0b440a80-b967-11ea-8138-4c40e6220edd.gif)

Using Space key instantly closes the prompt and does the replace all automatically without any action being done on the confirmation prompt. I will look into fixing this probably in the next week.